src/symbolize/gimli/coff.rs

```diff
@@ -1,6 +1,7 @@
 use super::{Context, Mapping, Path, Stash, Vec};
 use core::convert::TryFrom;
 use object::pe::{ImageDosHeader, ImageSymbol};
+use object::read::coff::ImageSymbol as _;
```

Comment:
It seems we don't need this line to bump object.

Reply:
It's strange, it breaks in a different way if I remove that. I think I'll close this PR, since I can't really test it on Windows.

Reply:
You do need this line, but you also need to update https://github.com/rust-lang/backtrace-rs/blob/b3e5bb857773fcc6fc8247374264bd4c8acd5387/crates/as-if-std/Cargo.toml#L22

Reply:
@lnicola 
This commit will make `rustc` failing in `mingw-check`:

https://github.com/rust-lang/rust/actions/runs/5407279353/jobs/9825113481

<img width="777" alt="image" src="https://github.com/rust-lang/backtrace-rs/assets/230646/13a1ce2d-3061-4027-866f-6f83efc21481">

Reply:
@chenyukang If you're bumping the `backtrace` version in `libstd`, then you also need to bump the `object` version.

Reply:
oh, sorry, I missed the previous comments.
Got it, thanks!